### DISCUSSION TITLE
X400 dac+ampli hat could working?

### DISCUSSION THREAD

**Discussion by Iorobyy11** [probonopd/MiniDexed#305] [General]
Hello i would ti bought this hat 
It’s a dac based on pcm5122 + amplifier 
http://www.suptronics.com/miniPCkits/x400_v3.0.html

this will work on minidexed ?

**Comment by diyelectromusic** [comment]
It looks like it is using the standard GPIO for I2S so I'd say your chances are pretty good for the DAC signal itself, but I see that the PCM5122 requires some kind of additional configuration over I2C or SPI, so I don't know about that, although it looks like the circle environment might support it, so you might be ok...

It definitely won't recognise the IR on GPIO26 or AMP mute on GPIO22 so you have to configure your pin usage will to keep those free to avoid weird IO clashes...
